### DISCUSSION TITLE
Error msg after trying to commit to a local branch

### DISCUSSION THREAD

**Discussion by tsironis13** [taiga-family/taiga-ui#9369] [General]
<img width="517" alt="image" src="https://github.com/user-attachments/assets/13ff28bf-811e-45f9-b321-90332ebcf9d3">

I am getting this error after forking the https://github.com/taiga-family/taiga-ui repo and trying to commit to a new local branch created.

**Comment by waterplea** [comment]
Closing this as you have opened an issue:
https://github.com/taiga-family/taiga-ui/issues/9372
PR is coming

**Reply by splincode** [reply]
@tsironis13 hi, check it please

**Reply by tsironis13** [reply]
Thanks, it works. I created the PR.
